### PR TITLE
docs(time-series-search): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-556.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+  gearbox: {
+    selectColor: 'red',
+    white: '#fff',
+  },
+};
+```
+
+<Preview>
+  <Story name="With Theme" >
+    <ThemeProvider theme={ExampleTheme}><TimeseriesSearch onTimeserieSelectionChange={onTimeserieSelectionChange}/></ThemeProvider>
```
Comment (mrtraser) [inline src/components/TimeseriesSearch/stories/basic.story.mdx]: alignment

**Comment by mrtraser** [comment]
`Show Root Asset Select` doesn't work properly, it doesn't fetch root assets for asset select. Pls, fix it

File: src/components/TimeseriesSearch/stories/basic.story.mdx (hunk 1 above)
Comment (amevck) [reply src/components/TimeseriesSearch/stories/basic.story.mdx]: This should be kept inline for showing better code snippets in docs.

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-556.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/stories/basic.story.mdx (hunk 1 above)
Comment (mrtraser) [reply src/components/TimeseriesSearch/stories/basic.story.mdx]: got it

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-556.gearboxjs.preview.cogniteapp.com

**Pull request merged**